### ISSUE TITLE
[bug] Fix UI

### ISSUE THREAD

**Issue by dklassic** [dklassic/GDC-transcript#85]
![image](https://github.com/user-attachments/assets/967d9ab0-9bbb-45cc-a441-bc98c1adca45)

Moving from MaterialTailwind to pure Tailwind breaks the visual, need to be fixed.

**Comment by Bri4tokyo** [comment]
Yes, it is working now. I tried a few videos, and they are all good. Thank you so much for fixing it!

**Comment by dklassic** [comment]
Finally got time to circle back and fix the UI fully in https://github.com/dklassic/GDC-transcript/commit/aa864490ba81ae02da84fe769042ea938e3a73a1 , should all be working now.
